README.md

```diff
@@ -32,3 +32,65 @@ How to build the SDK
  * datacapturing/build/outputs/aar/datacapturing-cyface-full-release.aar
  * synchronization/build/outputs/aar/synchronization-cyface-full-release.aar
 * For more details see: https://stackoverflow.com/a/27565045/5815054
```

Comment:
OH. I guess you could remove the stuff up here. It is not required anymore, is it?